build/test-pet-clinic/Dockerfile

```diff
@@ -11,4 +11,9 @@ RUN springBootUtility thin \
 FROM ${IMAGE}
 COPY --chown=1001:0 server.xml /config/server.xml
 COPY --from=staging /staging/lib.index.cache /lib.index.cache
-COPY --from=staging /staging/myThinApp.jar /config/dropins/spring/myThinApp.jar
\ No newline at end of file
+COPY --from=staging /staging/myThinApp.jar /config/dropins/spring/myThinApp.jar
+
+# Create a new SCC layer, args are layer size and number of iterations to run to populate it
+RUN populate_scc.sh 4m 3 \
```

Comment:
since this a test application, we should just call `configure.sh` here

Reply:
Both done: https://github.com/OpenLiberty/ci.docker/pull/117/commits/89e4bd3a1f035ee8381721917871434bb97a05e9.

Reply:
Both done: https://github.com/OpenLiberty/ci.docker/pull/117/commits/89e4bd3a1f035ee8381721917871434bb97a05e9.